packages/test/test-version-utils/src/compatUtils.ts

```diff
@@ -52,98 +46,37 @@ import { getRequestedVersion } from "./versionUtils.js";
-	optionsArg: IContainerRuntimeOptionsInternal = {
-		summaryOptions: {
-			summaryConfigOverrides: {
-				...DefaultSummaryConfiguration,
-				...{
-					initialSummarizerDelayMs: 0,
-				},
-			},
-		},
-	},
-	driverType: TestDriverTypes,
-) {
-	let options = { ...optionsArg };
-
-	// No test fails with this option, it allows us to validate properly expectations and
-	// implementation of services
-	options.loadSequenceNumberVerification = "close";
-
-	const compressorDisabled: ICompressionRuntimeOptions = {
-		minimumBatchSizeInBytes: Number.POSITIVE_INFINITY,
-		compressionAlgorithm: CompressionAlgorithms.lz4,
-	};
-
-	// These is the "maximum" config.
-	const {
-		compressionOptions = options.enableGroupedBatching === false
-			? disabledCompressionConfig
-			: {
-					minimumBatchSizeInBytes: 200,
-					compressionAlgorithm: CompressionAlgorithms.lz4,
-				},
-		enableGroupedBatching = true,
-		enableRuntimeIdCompressor = "on",
-		// Some t9s tests timeout with small settings. This is likely due to too many ops going through.
-		// Reduce chunking cut-off for such tests.
-		chunkSizeInBytes = driverType === "local" ? 200 : 1000,
-	} = options;
-
-	if (version.startsWith("1.")) {
-		options = {
-			// None of these features are supported by 1.3
-			compressionOptions: disabledCompressionConfig,
-			enableGroupedBatching: false,
-			enableRuntimeIdCompressor: undefined,
-			// Enable chunking.
-			// We need to ensure that 1.x documents (that use chunking) can still be opened by 2.x.
-			// This options does nothing for 2.x builds as chunking is only enabled if compression is enabled.
-			chunkSizeInBytes,
-			...options,
-		};
-	} else if (version.startsWith("2.0.0-rc.1.")) {
-		options = {
-			compressionOptions: compressorDisabled, // Can't use compression, need https://github.com/microsoft/FluidFramework/pull/20111 fix
-			enableGroupedBatching,
-			enableRuntimeIdCompressor: undefined, // it was boolean in RC1, switched to enum in RC2
-			chunkSizeInBytes: Number.POSITIVE_INFINITY, // disabled, need https://github.com/microsoft/FluidFramework/pull/20115 fix
-			...options,
-		};
-	} else if (version.startsWith("2.0.0-rc.2.")) {
-		options = {
-			compressionOptions: compressorDisabled, // Can't use compression, need https://github.com/microsoft/FluidFramework/pull/20111 fix
-			enableGroupedBatching,
-			// control over schema was generalized in RC3 - see https://github.com/microsoft/FluidFramework/pull/20174
-			// IdCompressor settings moved around - can't enable them across versions without tripping on asserts
-			enableRuntimeIdCompressor: undefined,
-			chunkSizeInBytes: Number.POSITIVE_INFINITY, // disabled, need https://github.com/microsoft/FluidFramework/pull/20115 fix
-			...options,
-		};
-	} else {
-		// "2.0.0-rc.3." ++
-		options = {
-			compressionOptions,
-			enableGroupedBatching,
-			chunkSizeInBytes,
-			enableRuntimeIdCompressor,
-			...options,
-		};
+function getMinVersionForCollab(
+	containerRuntimeVersion: string,
```

Comment:
Maybe use `runtimeVersion` and `runtimeVersionForLoading` for consistency with what we are calling the Runtime layer.